src/edu/washington/escience/myria/daemon/MyriaDriverLauncher.java

```diff
@@ -229,55 +228,60 @@ private static void logStartupMessage(Class<? extends Name<?>>[] classes, Inject
       final String doc = annotation.doc();
       final String defaultVal;
       {
-        if (!annotation.default_value().equals(NamedParameter.REEF_UNINITIALIZED_VALUE))
+        if (!annotation.default_value().equals(NamedParameter.REEF_UNINITIALIZED_VALUE)) {
           defaultVal = annotation.default_value();
-        else if (!annotation.default_class().equals(Void.class))
+        } else if (!annotation.default_class().equals(Void.class)) {
           defaultVal = annotation.default_class().getSimpleName();
-        else if (annotation.default_values().length > 0)
+        } else if (annotation.default_values().length > 0) {
           defaultVal = Arrays.toString(annotation.default_values());
-        else if (annotation.default_classes().length > 0) {
-          Class<?>[] default_classes = annotation.default_classes();
-          String[] sss = new String[default_classes.length];
-          for (int i1 = 0; i1 < default_classes.length; i1++) {
-            sss[i1] = default_classes[i1].getSimpleName();
+        } else if (annotation.default_classes().length > 0) {
```

Comment:
I think this code could be shortened/clarified quite a bit with some Guava/Java 8 constructs (untested):
```
final String[] classNames = Arrays.stream(annotation.default_classes()).map(c -> c.getSimpleName()).toArray();
final String classNameMsg = String.format("[%s]", Joiner.on(", ").join(classNames));
```